src/_guides/language/extension-methods.md

```diff
@@ -0,0 +1,246 @@
+---
+title: Extension methods
+description: Learn how to add to existing APIs.
+---
+<?code-excerpt replace="/ *\/\/\s+ignore_for_file:[^\n]+\n//g; /([A-Z]\w*)\d\b/$1/g; /\b(main)\d\b/$1/g"?>
+
+Extension methods, introduced in Dart 2.7,
+are a way to add functionality to existing libraries.
+You might not ever need to _use_ extension methods, much less _create_ them.
```

Comment:
I think this is an overstatement. I'd be very surprised if the majority of Dart developers were using at least one extension methods just a few months from now. We're even discussing adding a few extensions directly to the Flutter framework (as that would be non-breaking as opposed to changing core classes)